components/org.wso2.identity.webhook.common.event.handler/pom.xml

```
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+  ~ Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
```

```suggestion
  ~ Copyright (c) 2024-2025, WSO2 LLC. (http://www.wso2.com).
```
Add year range when changing the year of an older file. Check other places.

---

Changed